rpm/Makefile

```diff
@@ -1,22 +1,24 @@
-all: source
-		for dist in epel-7-x86_64 \
-				    fedora-27-x86_64 \
-				    fedora-28-x86_64 \
-				    fedora-29-x86_64 \
-					fedora-30-x86_64 ; do \
-				mock -r $$dist \
-				     --buildsrpm \
-					 --source=brave-keyring-source.tar.gz \
-					 --spec=brave-keyring.spec \
-					 --rebuild \
-					 --resultdir=output ; \
-		done
+all: rpm-build
+build: source
+	mock -r fedora-30-x86_64 \
+		--source=brave-keyring-source.tar.gz \
+        --spec=brave-keyring.spec \
+		--rebuild \
+		--old-chroot \
+		--resultdir=output
 
 keys:
-		wget https://brave-browser-apt-release.s3.brave.com/brave-core.asc -O brave-keyring-1.3/etc/pki/rpm-gpg/RPM-GPG-KEY-brave
+		wget https://brave-browser-apt-release.s3.brave.com/brave-core.asc -O brave-keyring-1.4/etc/pki/rpm-gpg/RPM-GPG-KEY-brave
```

Comment:
The version number here is unfortunate.

Reply:
Definitely unfortunate. We should make this more generic.